homeassistant/components/velbus/__init__.py

```diff
@@ -80,6 +80,31 @@ def syn_clock(self, service=None):
 
     hass.services.async_register(DOMAIN, "sync_clock", syn_clock, schema=vol.Schema({}))
 
+    def set_memo_text(service):
+        """Handle Memo Text service call."""
+        memo_text = service.data[ATTR_MEMO_TEXT]
+        memo_text.hass = hass
+        try:
+            controller.get_module(service.data[ATTR_MODULE_ADDRESS]).set_memo_text(
+                memo_text.async_render()
+            )
+        except velbus.util.VelbusException as err:
+            _LOGGER.error("An error occurred: %s", err)
+
+    hass.services.async_register(
+        DOMAIN,
+        "set_memo_text",
```

Comment:
Please create a constant in const.py for the service name e.g. `SERVICE_SET_MEMO_TEXT`

Reply:
Fixed in https://github.com/home-assistant/core/pull/31222/commits/aa7594916cc5f01452dde4fc72971c0b8bb2a2d8